app/blog/page.tsx

```diff
@@ -1,45 +1,34 @@
 import Link from "next/link";
-import { formatDate, getBlogPosts } from "app/lib/posts";
+import {getBlogPosts } from "app/lib/posts";
+import {metaData} from "../config";
+import Image from "next/image";
+import {ScreenWideSection} from "../components/screen-wide-section";
+import React from "react";
+import {BlogFilterComponent} from "../components/blogFilter";
```

Comment:
For more readabilty I would reorder the imports. Usually is good to use:
- Packages
- Project wide imports
- local imports
```
import Link from "next/link";
import React from "react";
import Image from "next/image";
import {getBlogPosts } from "app/lib/posts";
import {metaData} from "../config";
import {ScreenWideSection} from "../components/screen-wide-section";
import {BlogFilterComponent} from "../components/blogFilter";
```